optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-api/pom.xml

```diff
@@ -32,4 +32,25 @@
   <name>OptaPlanner Workbench - Domain Editor API</name>
   <description>OptaPlanner Workbench - Domain API</description>
 
+
+  <dependencies>
+
+    <!-- dependencies added because of new illegal transitive dependency check -->
```

Comment:
+1 to remove all these comments. It should become common knowledge that if in your module you have an .java file with "import foo.bar" than you need to have a direct dependency on foo-bar.